src/lib/attribute-manager.js

```diff
@@ -81,10 +81,13 @@ export default class AttributeManager {
     this._add(attributes, updaters);
   }
 
-  // Marks an attribute for update
+  /* Marks an attribute for update
+   * @param {string} attributeName: attribute or accessor name
+   */
   invalidate(attributeName) {
     const {attributes} = this;
-    const attribute = attributes[attributeName];
+    const attribute = attributes[attributeName] ||
+      Object.values(attributes).find(attr => attr.accessor === attributeName);
```

Comment:
This happens quite frequently. I believe we discussed pre-calculating a map to avoid searches here?

At least would like to see a comment mentioning this as a TODO or possible concernt.